features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/ImageButtonMapper.kt

```diff
@@ -39,22 +43,58 @@ internal class ImageButtonMapper(
         val (shapeStyle, border) = view.background?.resolveShapeStyleAndBorder(view.alpha)
             ?: (null to null)
 
+        // if the drawable has no width/height then there's no point trying to get a bitmap
+        // we willl still render a shape of the proper dimensions
+        if (drawable.intrinsicWidth > 0 && drawable.intrinsicHeight > 0) {
+            val imageWireframe = resolveImageWireframe(
+                view,
+                id,
+                bounds,
+                shapeStyle,
+                border,
+                drawable
+            )
+            wireframes.add(imageWireframe)
+        } else {
```

Comment:
This raises the question - is it better not to return a wireframe at all in this case, or should we return a shape with the bounds of the view?

Reply:
we should not return as it will not have any visible effect in the player.